cmd/server/main.go

```diff
@@ -312,6 +312,7 @@ func realMain(ctx context.Context) error {
 		userSub.Handle("", userController.HandleCreate()).Methods("POST")
 		userSub.Handle("/new", userController.HandleCreate()).Methods("GET")
 		userSub.Handle("/import", userController.HandleImport()).Methods("GET")
+		userSub.Handle("/import/userbatch", userController.HandleImportBatch()).Methods("POST")
```

Comment:
This is already under `/users` so having `user` in the route feels weird. Can we just use `/import` with a different verb instead?

```go
userSub.Handle("/import", usersController.HandleImport()).Methods("GET", "POST")
```